tensorflow_quantum/python/util.py

```diff
@@ -344,6 +345,86 @@ def _symbols_in_op(op):
         "tfq.util.get_supported_gates().")
 
 
+def is_expression_approx_eq(exp_1, exp_2, atol):
```

Comment:
Can we turn this into `_is_expression_approx_eq` since it appears we only want people using `is_gate_approx_eq` ?

Reply:
Done